.gitlab-ci.yml

```diff
@@ -108,7 +108,7 @@ format:
 # Documentation history
 # --------------------------------------
 
-generator:gource:
+gource:
```

Comment:
why?

Reply:
I didn't get the semantics of the generator: prefix (there's none I guess?) so I just removed it.

Reply:
no there is 😄 it is a substage of `generator`

Reply:
but there is no generator anymore? :D

Reply:
there used to be I think - I'd suggest to leave it to easily extend later on if necessary